readme.md

```diff
@@ -182,6 +182,7 @@ If you have used any of these styleguides, please add a comment if you can! The
 * [aturon](https://aturon.github.io)
 
 ## Swift
+* [AirBnB](https://github.com/airbnb/swift)
```

Comment:
Minor spelling:
```suggestion
* [Airbnb](https://github.com/airbnb/swift)
```

Reply:
@hugovk thank you for your review
I just following `JavaScript` and `Ruby` already using `AirBnB` 🤔

Reply:
Good point, please could you update those too?

Reply:
@hugovk I updated 👻